lang/de/t.php

```diff
@@ -587,6 +589,12 @@
 			"page_title" => ":participantName",
 			"really_delete_observation" => "Willst du diese Beobachtung wirklich löschen?",
 			"title" => "TN Details",
+			"shown_observations" => ":filtered von :total Beobachtungen angezeigt",
```

Comment:
Was mir erst jetzt auffällt ist dass es eigentlich cool wäre, die Filter in einem Satz zusammenzufassen. Dann könnten wir uns vielleicht auch den Help Text sparen, auch wenn der meiner Meinung nach jetzt nicht störend ist.
Irgendwas im Stil von: 
> 3 von 12 Beobachtungen von Bari im Block XY mit Anforderung(en) "blabla bla blabla blablabla" oder "lorem ipsum dolor sit..." werden angezeigt.

Für das zusammenfassen einer "Oder"-verknüpften Aufzählung als Satz gibt es [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat).

Aber können wir im Moment auch mal sein lassen.